cs-http-client/src/main/java/io/cloudslang/content/httpclient/build/conn/SSLConnectionSocketFactoryBuilder.java

```diff
@@ -98,9 +98,12 @@ public SSLConnectionSocketFactory build() {
         } else {
             try {
                 //need to override isTrusted() method to accept CA certs because the Apache HTTP Client ver.4.3 will only accepts self-signed certificates
-                KeyStore keyStore = createKeyStore(new URL("file:" + javaKeystore), changeit);
+                KeyStore keyStore = createKeyStore(new URL("file:" + keystore), keystorePassword);
+                sslContextBuilder.loadKeyMaterial(keyStore, keystorePassword.toCharArray());
 
-                sslContextBuilder.loadTrustMaterial(keyStore, new TrustSelfSignedStrategy() {
+                String internalJavaKeystoreUri = "file:" + System.getProperty("java.home") + "/lib/security/cacerts";
```

Comment:
reuse the `javaKeystore` from line 72 instead of `System.getProperty("java.home") + "/lib/security/cacerts"`